lib/rules/sotd/process-document.js

```diff
@@ -9,23 +9,23 @@ exports.check = function (sr, done) {
     ,   BOILERPLATE_SUFFIX = ' W3C Process Document.'
     ;
     var sotd = sr.getSotDSection()
-    ,   proc = "1 March 2019"
+    ,   proc = "15 September 2020"
     ,   boilerplate = BOILERPLATE_PREFIX + proc + BOILERPLATE_SUFFIX
-    ,   procUri = "https://www.w3.org/2019/Process-20190301/"
-    ,   deprecatedProc = "1 February 2018"
+    ,   procUri = "https://www.w3.org/2020/Process-20200915/"
+    ,   deprecatedProc = "1 March 2019"
     ,   deprecatedBoilerplate = BOILERPLATE_PREFIX + deprecatedProc + BOILERPLATE_SUFFIX
-    ,   deprecatedUri = 'https://www.w3.org/2018/Process-20180201/'
+    ,   deprecatedUri = 'https://www.w3.org/2019/Process-20190301/'
     ,   deprecatedAllowed
     ;
 
     sr.transition({
-        from:         new Date('2019-03-01')
-    ,   to:           new Date('2019-03-31')
+        from:         new Date('2020-09-14')
```

Comment:
In our case, we don't need to use the transition function as we don't have a transition period to switch processes.
The rule can be simplified (pseudo-code):
```
if (docDate < '2020-09-15') {
    proc = "1 March 2019"
    deprecatedUri = 'https://www.w3.org/2019/Process-20190301/'
} else {
    proc = "15 September 2020"
    procUri = "https://www.w3.org/2020/Process-20200915/"
}
```

No need to worry about deprecated process documents as we simply don't allow them.

Reply:
addressed in https://github.com/w3c/specberus/pull/954/commits/002ba22e7f19635237dbac24f02d2e41bb9206ba